deploy/kubernetes/tracee.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: tracee
```

Comment:
can you please rename the labels according to https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels?